charts/fulcio/Chart.yaml

```diff
@@ -5,7 +5,7 @@ description: |
 
 type: application
 
-version: 1.0.0
+version: 1.1.0
```

Comment:
I would bump this to 2.x.x and mention this release will be a breaking change to existing users.

Reply:
Thanks for the quick feedback!
You're right. I didn't consider the breaking consequences of changing the if condition.
I pushed the change just now.